packages/xo-server/src/xo-mixins/resource-sets.js

```diff
@@ -26,7 +26,7 @@ const VM_RESOURCES = {
   vms: true,
 }
 
-const computeVmResourcesUsage = vm => {
+const computeVmServerResourcesUsage = vm => {
```

Comment:
Do you mean by this rename that the computation is specific to a server?

Reply:
Please also fix the commit message.

Reply:
It means that it computes the resources used on the hosts (CPU, disk, ...), not in XO (IP pools). Maybe it's still not clear enough, though.